core/lib/generators/solidus/install/install_generator.rb

```diff
@@ -202,17 +187,16 @@ def install_frontend
       end
     end
 
-    def run_bundle_install_if_needed_by_plugins
-      @plugins_to_be_installed.each do |plugin_name|
-        gem plugin_name
-      end
+    def install_authentication_plugin
+      return unless @with_authentication
 
-      BundlerContext.bundle_cleanly { run "bundle install" } if @plugins_to_be_installed.any?
-      run "spring stop" if defined?(Spring)
+      install_plugin(plugin_name: 'solidus_auth_devise', plugin_generator_name: 'solidus:auth:install')
+    end
 
-      @plugin_generators_to_run.each do |plugin_generator_name|
-        generate "#{plugin_generator_name} --skip_migrations=true"
-      end
+    def install_payment_method_plugin
+      return unless @payment_method_gem_name
+
+      install_plugin(plugin_name: @payment_method_gem_name, plugin_generator_name: "#{@payment_method_gem_name}:install")
```

Comment:
Acknowledged :) I would prefer to keep the current implementation, since it's good enough for now. Giving the generator name  would probably require storing it in the `PAYMENT_METHODS`. It's not a complicated change, but at the same time, it's still a change that we can postpone making.

Reply:
Agree 🙂